package.json

```diff
@@ -33,7 +33,9 @@
   },
   "prettier": {
     "printWidth": 140,
-    "singleQuote": true
+    "singleQuote": true,
+    "trailingComma": "none",
+    "arrowParens": "avoid"
```

Comment:
`trailingComma` and `arrowParens` are enforced in the new version.
With their default values, formatting `Facet.ts` caused a lot of change. With these two rules, formatting `Facet.ts` causes no changes.
It may cause some changes elsewhere, but I imagine not as much.